common/djangoapps/course_about/data.py

```diff
@@ -0,0 +1,76 @@
+"""Data Aggregation Layer for the Course About API.
+This is responsible for combining data from the following resources:
+* CourseDescriptor
+* CourseAboutDescriptor
+"""
+import logging
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from course_about.serializers import serialize_content
+from course_about.errors import CourseNotFoundError
+from xmodule.modulestore.django import modulestore
+from xmodule.modulestore.exceptions import ItemNotFoundError
+from util.memcache import safe_key
+from django.core.cache import get_cache
+from django.conf import settings
+
+
+log = logging.getLogger(__name__)
+
+ABOUT_ATTRIBUTES = [
+    'effort',
+]
+
+
+def get_course_about_details(course_id):  # pylint: disable=unused-argument
+    """
+    Return course information for a given course id.
+    First its checks the default cache for given course id if its exists then returns
+    the course otherwise it get the course from module store and set the cache.
+    By default cache expiry set to 5 minutes.
+
+    Args:
+        course_id(str) : The course id to retrieve course information for.
+
+    Returns:
+        Serializable dictionary of the Course About Information.
+
+    Raises:
+        CourseNotFoundError
+    """
+    try:
+        cache = get_cache("default")
+        course_key = CourseKey.from_string(course_id)
+        key_prefix = getattr(settings, 'COURSE_INFO_API_CACHE_PREFIX', 'course_info_api_')
+        cache_key = safe_key(course_id, key_prefix,'')
+        course_info = cache.get(cache_key)
```

Comment:
1) Instead of getting the cache by name, you can just do this:
```
from django.core.cache import cache
```
which will always be the default cache.

2) `safe_key` is already installed as the cache's `KEY_FUNCTION`, so it should be called automatically by Django.  You don't need to call it explicitly here.

3) I don't think it's worth making `COURSE_INFO_API_CACHE_PREFIX` a setting -- this isn't something ops is going to need to override.  Instead, you can make it a top-level constant in this module.